### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by Baranovsky1986**
add  file filters with class RegisterFilter

### CODE REVIEW

File: data_ocean/filters.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        fields=(
+            ('id', 'name', 'source_last_update')
+        )
+    )
+
+    class Meta:
+        model = Register
+        fields = {
+            'name': ['icontains'],
+            'source_register_id': ['exact'],
```
Comment (roman-tiukh) [inline data_ocean/filters.py:16]: SearchFilter, we need SearchFilter

File: data_ocean/filters.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            'source_last_update': ['exact', 'gt', 'lt'],
```
Comment (roman-tiukh) [inline data_ocean/filters.py:17]: we dont need datetime filter, we need date filter...

Comment (roman-tiukh) [reply data_ocean/filters.py:17]: use DateFromToRangeFilter and DateFilter

File: data_ocean/filters.py (hunk 1 above)
Comment (roman-tiukh) [reply data_ocean/filters.py:16]: https://www.django-rest-framework.org/api-guide/filtering/#searchfilter

Comment (Baranovsky1986) [reply data_ocean/filters.py]: added SearchFilter

File: data_ocean/filters.py (hunk 2 above)
Comment (Baranovsky1986) [reply data_ocean/filters.py]: used DateFromToRangeFilter

**Comment by Baranovsky1986** [comment]
> Any requirements??

added django-filter to requirements

**Pull request merged**